hazelcast-integration/pom.xml

```diff
@@ -24,6 +24,7 @@
         <module>spring-hibernate-2ndlevel-cache</module>
         <module>enterprise-session-replication</module>
         <module>spring-security</module>
+        <module>hibernate-jpa-2ndlevel-cache</module>
```

Comment:
this module name should be same with artifact name of corresponding module. currently maven throws null pointer exception while compiling.